en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,81 @@
+    - Logs
+    - Diagnostic data
+
+Now you are ready to detect anomalies and perform root cause analysis on the sample service.
     
-## Detect anomalies and perform root cause analysis
-You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latencies of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
-
-### Step 1: Detect the anomalies
+## Step 2: Detect anomalies and perform root cause analysis
 
-#### Analyze the throughput graph
-The throughput graph shows the throughput of requests per hour for a selected timestamp. 
+To detect anomalies of the service, you must trace its executions from the code statement level to the machine it runs in. You can analyze observability data and logs to troubleshoot and identify the root cause for the anomalies you detect. 
 
-With the sample data, you can observe spikes in the throughput graph.  You can find the root cause for this by learning how to analyze the throughput graph.
+### Analyze the throughput graph
+The throughput graph depicts successful and erroneous requests to the service during a selected time interval. You can analyze the spikes in the throughput graph to identify the root cause for requests that result in an error.
 
-Follow the steps below. 
+Follow this procedure to analyze the throughput graph:
 
-1. Hover over the throughput graph where you see spikes. You can observe the number of errors that occurred at the time. 
+1. Go to the throughput graph under the **Throughput & Latency** tab and hold the pointer over data points where you see spikes. You can observe the number of errors for the corresponding fluctuations.
     
     !!!tip
-        You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
+        You can expand the graph by clicking and dragging the cursor over the period you want to drill down. The default graph displays data for a custom time range where the data points are at a lower granularity. This allows the effective detection of anomalies.
         
-2. Click the graph on a high point.
-3. Observe the logs in the logs panel.
-4. Note multiple logs are mentioning a connection error with the `hr-service`.
+2. Click on a data point where there is a spike and observe the log entries in the **Logs** pane.
  
-    ![Analyzing the throughput graph](../assets/img/observability/throughput-graph-analysis.png){.cInlineImage-full}
- 
-#### Analyze the latency graph
+    ![Analyze the throughput graph](../assets/img/observability/throughput-graph-analysis.png){.cInlineImage-full}
 
-The latency graph shows the latencies of each request.
+   Here, the log displays multiple entries indicating an error in connecting to the `hr-service`.
```

Comment:
We should remove this statement as it mentions a specific error.

Reply:
Addressed this.